client/src/components/AboutTheProduct/AboutTheProduct.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```

Comment:
I believe this is the home page? If so it need to replace client/src/pages/HomePage.js.   For now, I would place it there and call it HomePage-2.js